rust/main/utils/run-locally/src/starknet/utils.rs

```diff
@@ -0,0 +1,256 @@
+        match class.as_str() {
+            "contracts_merkle_tree_hook" => declared_classes.hpl_hook_merkle = class_hash,
+            "contracts_mailbox" => declared_classes.hpl_mailbox = class_hash,
+            "contracts_ism" => declared_classes.hpl_test_mock_ism = class_hash,
+            "contracts_pausable_ism" => declared_classes.hpl_ism_pausable = class_hash,
+            "contracts_hook" => declared_classes.hpl_test_mock_hook = class_hash,
+            "contracts_aggregation" => declared_classes.hpl_ism_aggregate = class_hash,
+            "contracts_message_recipient" => {
+                declared_classes.hpl_test_mock_msg_receiver = class_hash
+            }
+            "contracts_messageid_multisig_ism" => declared_classes.hpl_ism_multisig = class_hash,
+            "contracts_validator_announce" => declared_classes.hpl_validator_announce = class_hash,
+            "contracts_domain_routing_ism" => declared_classes.hpl_ism_routing = class_hash,
+            _ => println!("Unknown class: {}", class),
+        }
+    }
+
+    declared_classes
+}
+
+const VALIDATOR_ADDRESS: &str = "0x15d34AAf54267DB7D7c367839AAf71A00a2C6A65";
+const THRESHOLD: &str = "1";
+
+#[apply(as_task)]
+pub(crate) fn deploy_all(
+    cli: StarknetCLI,
+    deployer: String,
+    declarations: DeclaredClasses,
+    domain: u32,
+) -> Deployments {
+    // deploy ism - routing ism with empty routes
+    println!("Deploying routing ism");
+    let ism_routing = cli.deploy(declarations.hpl_ism_routing, vec![deployer.clone()]);
+
+    println!("Initializing routing ism");
+    cli.invoke(
+        ism_routing.clone(),
+        "initialize",
+        vec!["0".to_string(), "0".to_string()],
+    );
+
+    // deploy ism - multisig ism with no enrolled validators
+    println!("Deploying message id multisig ism");
+    let ism_multisig = cli.deploy(
+        declarations.hpl_ism_multisig,
+        vec![
+            deployer.clone(),
+            "1".to_string(),
+            VALIDATOR_ADDRESS.to_string(),
+            THRESHOLD.to_string(),
+        ],
+    );
+
+    // deploy pausable ism
+    println!("Deploying pausable ism");
+    let ism_pausable = cli.deploy(declarations.hpl_ism_pausable, vec![deployer.clone()]);
+
+    // deploy ism - aggregation
+    println!("Deploying aggregation ism");
+    let ism_aggregate = cli.deploy(
+        declarations.hpl_ism_aggregate,
+        vec![
+            deployer.clone(),
+            "2".to_string(),
+            ism_multisig.clone(),
+            ism_pausable,
+            THRESHOLD.to_string(),
+        ],
+    );
+    // deploy mock hook
+    println!("Deploying mock hook");
+    let mock_hook = cli.deploy(declarations.hpl_test_mock_hook, vec![]);
+
+    // deploy mailbox
+    println!("Deploying mailbox");
+    let mailbox = cli.deploy(
+        declarations.hpl_mailbox,
+        vec![
+            domain.to_string(),
+            deployer.clone(),
+            ism_multisig.clone(),
```

Comment:
should `ism_agregate` be the default mailbox ISM instead? For better testing coverage

Reply:
right